WorksheetC

```diff
@@ -0,0 +1,18 @@
+# Worksheet C
+1a)
+In this task of this procedure is to check if there is a duplicate in the list by taking each item in the list and checking it against every other one. If there is a duplicate it will return true and if not, it stops all the functions and returns false. In this algorithm, it takes the list and duplicates it to be able to check it against itself. 
+1b)
+The reason that this is a quadratic equation is that at the worst-case scenario it would have to cycle through both lists on every item in the list which is represented as n x n or n^2
```

Comment:
Not "both lists" -- there is only one list, but every pair of elements has to be considered hence n^2.